app/views/dashboard/_stripe_account_setup_banner.njk

```diff
@@ -1,12 +1,35 @@
 {% if permissions.stripe_account_details_update %}
   {% set stripeAccountSetup = account.accountSetupDetails %}
+
+  {% set panelModifierClass %}
+    {% if stripeAccountDetails.payouts_enabled === false %}
```

Comment:
We currently have the check `stripeAccountDetails.payouts_enabled === false` three times in this template as far as I can tell. I'd recommend setting this at the top of this context so that if the rules change the "how" of the template doesn't have to change. 

Something like: 
```njk
{% set stripeAccountIsRestricted = stripeAccountDetails.payouts_enabled === false %}
```

EDIT: 
Everything it might be useful to bring to the top
```njk
{% set stripeAccountIsRestricted = stripeAccount.payouts_enabled === false %}
{% set stripeAccountHasDeadline = stripeAccount.requirements.current_deadline %}
{% set connectorStripeJourneyIsComplete = stripeAccountSetup.bankAccount and stripeAccountSetup.vatNumber and stripeAccountSetup.companyNumber and stripeAccountSetup.responsiblePerson %}
```

Reply:
Done